src/document.js

```diff
@@ -233,7 +234,7 @@ const documentsMain = {
 			$(document.body).addClass('claro')
 			$(document.body).prepend(documentsMain.UI.container)
 
-			const urlsrc = getWopiUrl({ fileId, title, readOnly: false, closeButton: true, revisionHistory: !!Config.get('userId') })
+			const urlsrc = getWopiUrl({ fileId, title, readOnly: false, closeButton: !documentsMain.isPublic, revisionHistory: !documentsMain.isPublic })
```

Comment:
Might need extra handling as the close button would still be required if a file is opened in a shared folder.